### PR TITLE
Feat/sidebar

### PR DESCRIPTION
**Pull request by lizardK**
### Description of the Change

In order to improve loading performance the sidebar loads services in asynchronous way and lazyload items when user want to access it.

### Dependencies
Should be merged after : 
- [x] https://github.com/ovh-ux/manager/pull/778
- [x] https://github.com/ovh-ux/ng-ovh-sidebar-menu/pull/62

### References
MANAGER-2740

## Build branch 
https://github.com/ovh-ux/ovh-manager-web/tree/build/sidebar

### CODE REVIEW

File: client/app/css/less/common/navbar.less [hunk 1]
```diff
@@ -31,6 +31,10 @@
   }
 }
 
+.left-0 {
```
Comment (antleblanc) [inline client/app/css/less/common/navbar.less]: Is it still useful? I can't find where you need this helper.

File: client/app/dns-zone/dns-zone.controller.js [hunk 2]
```diff
@@ -71,21 +68,18 @@ angular.module('App').controller(
...
     loadDomain() {
       this.loading.domainsInfos = true;
-
-      this.$q
-        .all({
-          product: this.WucProducts.getSelectedProduct(true),
-          tabZone: this.Domain.getTabZoneDns(this.$stateParams.productId, 1),
-        })
-        .then(({ product, tabZone }) => {
+      this.Domain.getTabZoneDns(this.$stateParams.productId, 1)
```
Comment (antleblanc) [inline client/app/dns-zone/dns-zone.controller.js]: You can return this promise.

File: webpack.config.js [hunk 3]
```diff
@@ -83,15 +83,15 @@ module.exports = (env = {}) => {
...
-      components: glob.sync('./client/app/components/**/!(*.module).js'),
+      ].concat(glob.sync('./client/app/**/*.module.js'))
+        .concat(glob.sync('./client/app/components/**/!(*.module).js')),
     }, bundles, extras.length > 0 ? { extras } : {}),
     output: {
       path: path.resolve(__dirname, 'dist'),
       filename: '[name].[chunkhash].bundle.js',
     },
     resolve: {
+      symlinks: false,
```
Comment (antleblanc) [inline webpack.config.js]: This change must be amended before merging this pull request.

File: client/app/components/webApp.controller.js [hunk 4]
```diff
@@ -41,13 +36,21 @@ export default class WebAppCtrl {
       }
     };
 
-    this.webNavbar.getResponsiveLinks()
+    /* this.webNavbar.getResponsiveLinks()
```
Comment (marie-j) [inline client/app/components/webApp.controller.js]: You can remove if not used :)

File: client/app/dns-zone/dns-zone.controller.js (hunk 2 above)
Comment (JeremyDec) [inline client/app/dns-zone/dns-zone.controller.js]: Could `1` param be stored in a const please ? In order to see directly what is the purpose of this param

**Comment by ovh-cds** [comment]
CDS Report build-manager#1494.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✔ 
  * Build manager EU ✘  
  * Build manager US ✔ 
  * Tests ✘

**Pull request merged**